ipatests/conftest.py

```diff
@@ -169,3 +173,24 @@ def fin():
 
     request.addfinalizer(fin)
     return tempdir
+
+
+@pytest.fixture(scope='class')
+def ipa_ad_trust(mh):
```

Comment:
This fixture only applies to integration tests and depends on the multihost ``mh`` fixture. Please move it to ``ipatests/pytest_ipa/integration/__init__.py``.

Reply:
@tiran 
fixed